README.md

```diff
@@ -49,12 +46,9 @@ roles: {
 zone_id: "<enter region>"
 min_workers: "<min workers>"
 max_workers: "<max workers>"
-spark_version: "<spark version in cluster>"
-node_type_id : "<node type>"
+node_type_id: "<node type>"
 autotermination_minutes: "<time out>"
```

Comment:
Remove quotes

Reply:
Thanks for catching This seems to be missed though code changes was done